deploy/determined_deploy/aws/deployment_types/simple.py

```diff
@@ -34,6 +34,10 @@ class Simple(base.DeterminedDeployment):
         constants.cloudformation.SUBNET_ID_KEY,
         constants.cloudformation.SCHEDULER_TYPE,
         constants.cloudformation.PREEMPTION_ENABLED,
+        constants.cloudformation.CPU_IMAGE,
+        constants.cloudformation.GPU_IMAGE,
+        constants.cloudformation.CPU_IMAGE,
```

Comment:
This duplication isn't intentional, right?

Reply:
Yup - removed